.travis/check-manifest.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 ./scripts/create-manifest.py
+git status
+git diff
```

Comment:
these two lines are just checking that they both return 0?

Reply:
Oops, no. They were purely debugging to figure out why Travis was acting different. I've removed them.